ibm_cloud_sdk_core/base_service.py

```diff
@@ -33,23 +34,39 @@
 # import http.client as http_client
 # http_client.HTTPConnection.debuglevel = 1
 
-
 class BaseService(object):
-
+    """Abstract class for common functionality between generated services.
```

Comment:
Is this really an "abstract" class???

Reply:
Also, maybe a comment like "Common functionality shared by generated service classes." would be a little bit more clear?

Reply:
:+1: